### DISCUSSION TITLE
Research and design a new two-up view that does not do book view calculations.

### DISCUSSION THREAD

**Discussion by mejackreed** [ProjectMirador/mirador#3461] [Ideas]
It may be useful for Mirador users to have a two-up view within an object where the second canvas is not the back of the first canvas. 

An example here is the scanning and representation of a booklike thing was scanned published in a non-IIIF `paged` compatible way. Or maybe there are two images next two each other that would be useful to shown together?

This ticket indicates that we need better use cases to support what this should be. Also design will need to go in supporting these cases.

**Comment by blalbrit** [comment]
Thanks @mejackreed - I'm going to list here a subset of materials where two-up viewing may be desired, for items that are not registered as "book" in the Stanford Digital Repository. The links will not be accessible to those outside of the Stanford domain, but Stanford developers can pull from these for representative samples.

Of the materials registered as "image" objects in the SDR, we have:
* Sheet music collections where single-page or facing-page views may be desirable (https://argo.stanford.edu/catalog?f%5Bis_member_of_collection_ssim%5D%5B%5D=info%3Afedora%2Fdruid%3Azw287hd4612)
* Correspondence collections where viewing front and back of a letter is desirable (https://argo.stanford.edu/catalog?f%5Bis_member_of_collection_ssim%5D%5B%5D=info%3Afedora%2Fdruid%3Anx585yw5390)
* Correspondence collections where envelope and letters are both recorded and front/back viewing of the letter in the viewer may be desirable in addition to single-image viewing (https://argo.stanford.edu/catalog?f%5Bis_member_of_collection_ssim%5D%5B%5D=info%3Afedora%2Fdruid%3Ays767tp0800)
* medieval legal documents where viewing front and back simultaneously can be helpful in addition to single-image viewing( https://argo.stanford.edu/catalog?f%5Bis_member_of_collection_ssim%5D%5B%5D=info%3Afedora%2Fdruid%3Adm390bc4811)
* papyrus collections where viewing front/back is desirable alongside single-image viewing (https://argo.stanford.edu/catalog?f%5Bcontent_type_ssim%5D%5B%5D=image&f%5Bnonhydrus_collection_title_ssim%5D%5B%5D=Classics+Papyri)
* manuscript fragment collections where side-by-side front/back can be very useful in addition to single-image view (https://argo.stanford.edu/catalog?f%5Bcontent_type_ssim%5D%5B%5D=image&f%5Bnonhydrus_collection_title_ssim%5D%5B%5D=Medieval+fragments+study+collection%2C+11th-16th+cent)
* postcard collections where front/back side-by-side views can be useful alongside single-image viewing (https://argo.stanford.edu/catalog?f%5Bis_member_of_collection_ssim%5D%5B%5D=info%3Afedora%2Fdruid%3Amr709rx9041)
* coin collections where viewing both sides of the coin can be very helpful for researchers, alongside single-image view (https://argo.stanford.edu/catalog?f%5Bcontent_type_ssim%5D%5B%5D=image&f%5Bnonhydrus_collection_title_ssim%5D%5B%5D=R.+M.+Row+collection+of+Roman+Imperial+coins)
* jazz scores where single and facing-page views are both desirable for a user (https://argo.stanford.edu/catalog?f%5Bcontent_type_ssim%5D%5B%5D=image&f%5Bnonhydrus_collection_title_ssim%5D%5B%5D=Edward+Lawless+Traditional+Jazz+Photograph+Collection)

I raise these use-cases not to argue against bringing Mirador into line with IIIF spec, but to suggest that reducing viewing modalities based on a rich variety of content types might be less than desirable from a user perspective. I have provided use-cases that I know about in the current digitized materials at Stanford (and have not been exhaustive), it is likely that more curatorial colleagues, digitization experts, and other institutions can add to this list.
